Packs/FidelisEndpoint/Integrations/FidelisEndpoint/FidelisEndpoint.py

```diff
@@ -0,0 +1,1802 @@
+    column = args.get('column')
+    value = args.get('value')
+    operator = args.get('operator')
+    limit = args.get('limit')
+    additional_filter_string = args.get('additional_filter')
+    additional_filter = None
+    if additional_filter_string:
+        additional_filter_split = additional_filter_string.split()
+        if len(additional_filter_split) == 3:
+            additional_filter = {
+                'filterType': 'criteria',
+                'column': additional_filter_split[0],
+                'operator': additional_filter_split[1],
+                'value': additional_filter_split[2]
+            }
+        else:
+            raise Exception('Make sure that the additional_filter argument is in valid format.\n '
+                            'For Example: pid = 1234')
+
+    contents = []
+    context = []
+    headers = ['PID', 'EndpointName', 'User', 'ProcessStartTime', 'LocalIP', 'LocalPort', 'RemoteIP', 'RemotePort',
+               'ParentID', 'EventType']
+    response = client.query_events(limit, start_time, end_time, logic, column, value, entity_type, operator,
+                                   additional_filter)
+    if not response.get('success'):
+        raise Exception(response.get('error'))
+    res = response.get('data', {})
+    events = res.get('events', [])
+    if not events:
+        return f'No events were found', {}, {}
+
+    for event in events:
+        contents.append({
+            'EndpointName': event.get('endpointName'),
+            'EventType': event.get('eventType'),
+            'ParentID': event.get('parentId'),
+            'PID': event.get('pid'),
+            'User': event.get('user'),
+            'ProcessStartTime': event.get('processStartTime'),
+            'LocalIP': event.get('localIP'),
+            'LocalPort': event.get('localPort'),
+            'RemoteIP': event.get('remoteIP'),
+            'RemotePort': event.get('remotePort')
+        })
+
+        context.append({
+            'EventTime': event.get('eventTime'),
+            'EndpointName': event.get('endpointName'),
+            'EventType': event.get('eventType'),
+            'ParentID': event.get('parentId'),
+            'TargetID': event.get('targetId'),
+            'PID': event.get('pid'),
+            'ParentName': event.get('parentName'),
+            'Name': event.get('name'),
+            'Path': event.get('path'),
+            'Hash': event.get('hash'),
+            'User': event.get('user'),
+            'LocalIP': event.get('localIP'),
+            'LocalPort': event.get('localPort'),
+            'RemoteIP': event.get('remoteIP'),
+            'RemotePort': event.get('remotePort'),
+            'DnsQuestion': event.get('dnsQuestion'),
+            'DnsAnswer': event.get('dnsAnswer'),
+            'ProcessStartTime': event.get('processStartTime'),
+            'EventIndex': event.get('eventIndex'),
+            'IndexingTime': event.get('indexingTime'),
+            'EntityType': event
```

Comment:
if there will be no alerts fetched, we wont get in the loop, and `latest_alert_create_date` will be referenced before assignment
lets initalize `latest_alert_create_date = last_fetch_alert_time` before the loop